cmake/rapids-cmake.json

```diff
@@ -0,0 +1,154 @@
+{
```

Comment:
What is this file? Is it documented anywhere?

Reply:
I need to remove this. 

The intent is for `ci/checks/style.sh` to download this file from the rapids-cmake project ( https://github.com/rapidsai/rapids-cmake/blob/branch-21.08/cmake-format-rapids-cmake.json ).  Inside rapids-cmake this file is manually updated each time API is modified. 

The file is used by `cmake-format` so that it can do a better job formatting `rapids_` commands. For example this file tells `cmake-format` that the `rapids_cpm_find` command has an option `GIT_REPOSITORY` which requires a single value. Therefore `cmake-format` knows not to `GIT_REPOSITORY` and the argument onto separate lines